packages/client/lib/commands/CONFIG_SET.ts

```diff
@@ -1,5 +1,15 @@
-export function transformArguments(parameter: string, value: string): Array<string> {
-    return ['CONFIG', 'SET', parameter, value];
+export function transformArguments(
+    parameter: string,
+    value: string,
+    mulitipleParams?: Array<[parameter: string, value: string]>
+): Array<string> {
+    const args = ['CONFIG', 'SET', parameter, value];
+
+    if (mulitipleParams) {
+        mulitipleParams.forEach(pair => args.push(pair[0], pair[1]));
```

Comment:
should we destructure for clarity?
```suggestion
        mulitipleParams.forEach(([parameter, value]) => args.push(parameter, value));
```

Reply:
@Avital-Fine checkout my commit... :)